edi_purchase_base/models/supplier_price_list.py

```diff
@@ -30,6 +30,7 @@ class SupplierPriceList(models.Model):
         help="The price HT to purchase a product",
     )
     apply_date = fields.Date(readonly=True, required=True)
+    ean_code = fields.Char(string="Ean")
```

Comment:
can you call it `barcode` pls? as it is odoo standard for that field..

Also can you pls check travis, because it's failing